docs/bff-guide.md

```
@@ -0,0 +1,696 @@
+    "message": "name is required"
+  }
+}
+```
+
+---
+
+## Full Request Lifecycle
+
+Here's what happens for a typical `GET /api/v1/my-resources` request:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)